Apps/Inc/getter.h

```diff
@@ -0,0 +1,8 @@
+#define GETTER(type, name) \
+    type get_##name(void){ \
```

Comment:
Using `##` on its own doesn't do macro expansion, you need a second layer: https://stackoverflow.com/questions/12630548/c-expand-macro-with-token-pasting